### ISSUE TITLE
Missing alt text on logo

### ISSUE THREAD

**Issue by DavidIsaacBPS** [HHS/simpler-grants-gov#7972]
### Summary

The Simpler.grants.gov logo at the top of the page is an image and a link but has no alt text. This is a minor accessibility inconvenience.

### Reproducibility

I tried multiple times and saw the bug every time

### How to reproduce the bug

Hover over logo

### Browser

Chrome

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct
